doc/nrf/gs_programming.rst

```diff
@@ -86,6 +87,53 @@ You can download it from the following links:
    .. note::
    	In a multi-image build, this allows you to debug the source code of your application only.
 
+
+.. _gs_programming_manage:
+
+Managing projects with SES
+=========================
+
+Using SES it is possible to add/remove files to the application project using the SES.
+
+It is also possible to configure other settings for the application.
+
+In order to manage CMakeLists.txt files using SES - Nordic Edition, the tags
+
+    - ``# NORDIC SDK APP START``
+    - ``# NORDIC SDK APP END``
+
+must be surrounding the following CMake commands in the application CMakeLists.txt file.
+
+    - ``target_sources``
+    - ``target_compile_definitions``
+    - ``target_include_directories``
+    - ``target_compile_options``
```

Comment:
I assume this is just for the 'app' library?

If so, this should be stated.

One can define multiple libraries in the application CMakeLists.txt file.

Reply:
Good point.
To be made clearer.